{
  "id": "BIT-mlflow-2023-6974",
  "details": "A malicious user could use this issue to access internal HTTP(s) servers and in the worst case (ie: aws instance) it could be abuse to get a remote code execution on the victim machine.",
  "aliases": [
    "CVE-2023-6974",
    "GHSA-59v3-898r-qwhj"
  ],
  "modified": "2024-03-06T11:25:28.861Z",
  "published": "2024-03-06T10:56:37.402Z",
  "database_specific": {
    "cpes": [
      "cpe:2.3:a:lfprojects:mlflow:*:*:*:*:*:*:*:*"
    ],
    "severity": "Critical"
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/mlflow/mlflow/commit/8174250f83352a04c2d42079f414759060458555"
    },
    {
      "type": "WEB",
      "url": "https://huntr.com/bounties/438b0524-da0e-4d08-976a-6f270c688393"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "mlflow",
        "ecosystem": "Bitnami",
        "purl": "pkg:bitnami/mlflow"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2.9.2"
            }
          ]
        }
      ],
      "database_specific": {
        "source": "https://github.com/bitnami/vulndb/tree/main/data/mlflow/BIT-mlflow-2023-6974.json"
      },
      "severity": [
        {
          "type": "CVSS_V3",
          "score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:N/A:N"
        }
      ]
    }
  ],
  "schema_version": "1.6.0"
}